packages/jest-preset-default/scripts/setup-globals.js

```diff
@@ -51,6 +22,3 @@ global.window.localStorage = {
 
 // UserSettings global
 global.window.userSettings = { uid: 1 };
-
-// Mock jQuery
-global.window.jQuery = { holdReady() {} };
```

Comment:
We no longer use `holdReady` in the codebase, besides it isn't general enough to be here.